aii-ks/src/main/pan/quattor/aii/ks/schema.pan

```diff
@@ -75,50 +75,50 @@ type structure_ks_mail = {
     for user customization are under /system/ks/hooks/.
 }
 type structure_ks_ks_info = {
-    "ackurl"	: type_absoluteURI
-    "acklist"	? type_absoluteURI[]
+    "ackurl" : type_absoluteURI
+    "acklist" ? type_absoluteURI[]
     "auth" : string[] = list ("enableshadow", "passalgo=sha512")
     "bootloader_location" : string = "mbr"
     "bootloader_append" ? string
     "bootloader_password" ? string
     "bootdisk_order" ? string[] # From DESYs template
-    "clearmbr"	: boolean = true
+    "clearmbr" : boolean = true
     "enable_service" ? string[]
-    "enable_sshd"   : boolean = false
-    "clearpart"	? string []
-    "driverdisk"	? type_absoluteURI[]
+    "enable_sshd" : boolean = false
+    "clearpart" ? string []
+    "driverdisk" ? type_absoluteURI[]
     @{deprecated boolean. when defined, precedes value of mail/success.}
-    "email_success" ? boolean with {deprecated(0, "email_success is deprecated; use mail/success instead"); true;}
-    "firewall"	? structure_ks_ksfirewall
-    "installtype"	: string
+    "email_success" ? boolean with {deprecated(0, "email_success is deprecated; use mail/success instead"); true; }
+    "firewall" ? structure_ks_ksfirewall
+    "installtype" : string
     "installnumber" ? string
-    "lang"		: string = "en_US.UTF-8"
+    "lang" : string = "en_US.UTF-8"
     # TODO: how would you set --default=your_lang in ks?
     @{If you use more than one language, mark the default one with --default your_lang}
-    "langsupport"	? string [] = list ("en_US.UTF-8")
-    "logging"	? structure_ks_logging
-    "mouse"		? string
-    "mail"      : structure_ks_mail
-    "bootproto"	: string with match (SELF, "static|dhcp")
-    "keyboard"	: string = "us"
-    "node_profile"	: type_absoluteURI
-    "rootpw"	: string
+    "langsupport" ? string [] = list ("en_US.UTF-8")
+    "logging" ? structure_ks_logging
+    "mouse" ? string
+    "bootproto" : string with match (SELF, "static|dhcp")
+    "mail" : structure_ks_mail
```

Comment:
any particular reason these lines switched places?

Reply:
no. i probably moved the `mail` around a bit and then decided to put it back in the wrong place 😄 do you want me to change it?

Reply:
Doesn't really matter, just confuses the history slightly.